template_debian/appmenus_buster_gnome/netvm-whitelisted-appmenus.list

```diff
@@ -0,0 +1 @@
+gnome-terminal.desktop
```

Comment:
This should be `org.gnome.Terminal.desktop`. `gnome-terminal.desktop` doesn't exist anymore.

Reply:
Also, it may be just me, but sometimes launching gnome-terminal (with all its dependent gnome-related services) in netvm cause out of memory issues. For this reason I'd recommend xterm here, even though it's much less convenient.

Reply:
I copied from wheezy and not stretch which is the newer version. I fixed it and switched to xterm for netvm.